ppafm/cli/conv_rho.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/python
-# This is a sead of simple plotting script which should get AFM frequency delta 'df.xsf' and generate 2D plots for different 'z'
-
-import __main__ as main
-import matplotlib.pyplot as plt
 import numpy as np
 
 import ppafm as PPU
 import ppafm.fieldFFT as fFFT
 from ppafm import io
 
-# ======== Functions
+HELP_MESSAGE = f"""Use this program in the following way:
+ppafm-conv-rho -s <filename> -t <filename> [ -A <value> ] [ -B <value> ]
+Supported file fromats are:
+    * xsf
+"""
+
```

Comment:
I don't think this is needed. You can get a help message for the script by simply calling it with the `-h` option. E.g. `ppafm-conv-rho -h`.

Reply:
Alright. However, `generateElFF.py` does have the `HELP_MESSAGE` constant defined like the above. Is there any reason why `generateElFF.py` is special? Or should I remove the `HELP_MESSAGE`  from `generateElFF.py` too?

Reply:
I don't see any reason why `generateElFF.py` would need it either. If you need to print the help message from within the program, you can use the [`parser.print_help()`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.print_help) method. I would remove this from both of the scripts.